### PR TITLE
Write first tests

### PR DESCRIPTION
**Pull request by dbosen**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-cache
+          restore-keys: ${{ runner.os }}-npm-cache
+
+      - name: Setup environment variables
+        run: |
+          echo "::add-path::$HOME/.composer/vendor/bin"
+
+      - name: Get build environment
+        run: composer global require thunder/drupal-testing
```
Comment (chrfritsch) [inline .github/workflows/test.yml]: Before we require this, we have to set a custom composer path to the current directory to get the PR code

File: tests/module/module.info.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: Module
+type: module
+description: Drupal module with all sorts of tests, to test capabilities of the drupal-testing package.
+package: Drupal Testing
+core: 8.x
+core_version_requirement: ^8 || ^9
```
Comment (chrfritsch) [inline tests/module/module.info.yml]: Test modules don't need a core or core_version_requirement key

File: tests/module/module.libraries.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Custom module library for general purposes.
```
Comment (chrfritsch) [inline tests/module/module.libraries.yml]: I think this file is not needed

File: tests/module/module.info.yml (hunk 2 above)
Comment (dbosen) [reply tests/module/module.info.yml]: Why not?

File: .github/workflows/test.yml (hunk 1 above)
Comment (chrfritsch) [reply .github/workflows/test.yml]: Solved with env var, like in travis

File: tests/module/module.info.yml (hunk 2 above)
Comment (chrfritsch) [reply tests/module/module.info.yml]: Ah, I thought it's a real test module, but it's the module we want to test ;)

**Pull request merged**